server/service/src/sync/test/mod.rs

```diff
@@ -0,0 +1,184 @@
+// mod integration;
+mod pull_and_push;
+pub(crate) mod test_data;
+
+use super::translations::IntegrationRecords;
+use crate::sync::translations::PullUpsertRecord;
+use repository::*;
+use util::inline_init;
+
+#[derive(Clone)]
+pub(crate) struct TestSyncPullRecord {
+    /// Expected result for the imported data
+    pub(crate) translated_record: Option<IntegrationRecords>,
```

Comment:
yeah, private would cause compilation error (since field is used directly) and pub would cause warnings if/when we use latest manifest edition (since pub fields in pub(crate) struct don't make sense, i've updated other in that struct here: f857e35211458f31846634d31f0a74ec9b58a160 in https://github.com/openmsupply/open-msupply/issues/338